mdtk/fileio.py

```diff
@@ -201,9 +202,12 @@ def csv_to_df(csv_path, single_track=False, non_overlapping=False):
             track (int): the track of the note.
             pitch (int): the MIDI pitch of the note.
             dur (int): the duration of the note, in ms.
-        Sorting will be first by onset, then track, then pitch, then duration.
+            velocity (int, optional): the velocity of the note. Defaults to 100.
+        Sorting will be first by onset, then track, then pitch, then duration,
+        then velocity.
     """
     df = pd.read_csv(csv_path, names=NOTE_DF_SORT_ORDER)
+    df["velocity"] = df["velocity"].fillna(100).astype(int)
```

Comment:
instead of adding `100` with no explanation, please make this a global var `DEFAULT_VELOCITY_VALUE` or something.

Reply:
Yep, will do that